test/integration/index.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const bodyParser = require('body-parser');
+const express = require('express');
+const http = require('http');
+const logging = require('@google-cloud/logging')();
+
+// Application implementing the integration test spec at
+// https://github.com/GoogleCloudPlatform/runtimes-common/blob/master/integration_tests/README.md
+const app = express();
+
+app.use(bodyParser.json());
+app.get('/', (req, res) => {
+  res.status(200).send('Hello World!');
+});
+
+app.post('/logging', (req, res) => {
```

Comment:
I just changed the spec for the integration tests to support "standard" and "custom" logging: basically, standard is logging through the runtime's logging module (what you're doing here), and custom is using a client library to write a manual log message to Stackdriver. Check out the updated docs [here](https://github.com/GoogleCloudPlatform/runtimes-common/tree/master/integration_tests), and see how I did it in the Python app [here](https://github.com/GoogleCloudPlatform/python-runtime/blob/master/tests/integration/server.py#L58).